static/js/settings_display.js

```diff
@@ -90,7 +90,7 @@ function default_language_modal_post_render($container) {
     }
 }
 
-export function launch_default_language_setting_modal($container) {
+export function launch_default_language_setting_modal() {
     const html_body = render_dialog_default_language({
         language_list: get_language_list_columns(user_settings.default_language),
```

Comment:
This seems to be a bug, we would need to pass `realm_default_language` for notification language modal.

Reply:
I am not sure what you mean. Why do we need to pass it here? The handlebar doesn't seem to be using it.

Reply:
I mean that we should pass `realm_default_language` to `get_language_list_column` for notification language modal because we show the selected language in different color in the modal. You can reproduce it by changing notification lamguage to be something other than the user setting language and then open the modal again.

Reply:
Oh! good point.